fe/fe-core/src/main/java/com/starrocks/analysis/MultiListPartitionDesc.java

```diff
@@ -0,0 +1,163 @@
+            }
+        }
+        this.analyzeDuplicateValues(partitionColSize);
+        this.analyzeProperties(tableProperties);
+    }
+
+    private void analyzeProperties(Map<String, String> tableProperties) throws AnalysisException {
+        FeNameFormat.checkPartitionName(this.getPartitionName());
+
+        // copy one. because ProperAnalyzer will remove entry after analyze
+        Map<String, String> copiedTableProperties = Optional.ofNullable(tableProperties)
+                .map(properties -> Maps.newHashMap(properties))
+                .orElseGet(() -> new HashMap<>());
+        Map<String, String> copiedPartitionProperties = Optional.ofNullable(this.partitionProperties)
+                .map(properties -> Maps.newHashMap(properties))
+                .orElseGet(() -> new HashMap<>());
+
+        // The priority of the partition attribute is higher than that of the table
+        Map<String, String> allProperties = new HashMap<>();
+        copiedTableProperties.forEach((k, v) -> allProperties.put(k, v));
+        copiedPartitionProperties.forEach((k, v) -> allProperties.put(k, v));
+
+        // analyze data property
+        this.partitionDataProperty = PropertyAnalyzer.analyzeDataProperty(allProperties,
+                DataProperty.DEFAULT_DATA_PROPERTY);
+
+        // analyze replication num
+        this.replicationNum =
+                PropertyAnalyzer.analyzeReplicationNum(allProperties, FeConstants.default_replication_num);
+
+        // analyze version info
+        this.versionInfo = PropertyAnalyzer.analyzeVersionInfo(allProperties);
+
+        // analyze in memory
+        this.isInMemory =
+                PropertyAnalyzer.analyzeBooleanProp(allProperties, PropertyAnalyzer.PROPERTIES_INMEMORY, false);
+
+        // analyze tabletType
+        this.tabletType = PropertyAnalyzer.analyzeTabletType(allProperties);
+
+        // check unknown properties
+        if (copiedTableProperties.isEmpty() && !allProperties.isEmpty()) {
+            Joiner.MapJoiner mapJoiner = Joiner.on(", ").withKeyValueSeparator(" = ");
+            throw new AnalysisException("Unknown properties: " + mapJoiner.join(allProperties));
+        }
+    }
+
+    private void analyzeDuplicateValues(int partitionColSize) throws AnalysisException {
+        Set[] container = new TreeSet[partitionColSize];
+        for (List<String> values : multiValues) {
+            int duplicatedSize = 0;
+            for (int i = 0; i < values.size(); i++) {
+                if (container[i] == null) {
+                    container[i] = new TreeSet();
+                }
+                if (!container[i].add(values.get(i))) {
+                    duplicatedSize++;
+                }
+            }
+            if (duplicatedSize == partitionColSize) {
+                throw new AnalysisException(
+                        "(" + String.join(",", values) + ") size should be equal to partition column size ");
+            }
+        }
```

Comment:
Check logic has some problem. 
for example,
two partition columns (date and int),
partition values:
("2022-04-22", 1)
("2022-04-23", 2)
("2022-04-22", 2)